### PR TITLE
WIP 

### PR DESCRIPTION
**Pull request by vjgaur**
Fixing and Updating Mock

### CODE REVIEW

File: pallets/schema/src/mock.rs [hunk 1]
```diff
@@ -49,8 +54,8 @@ where
 	}
 }
 
-pub fn generate_schema_id<T: Config>(digest: &SchemaHashOf<T>) -> IdentifierOf {
-	let identifier: IdentifierOf = curi::generate(&(&digest).encode()[..], SCHEMA_PREFIX)
+pub fn generate_schema_id<T: Config>(digest: &SchemaHashOf<T>) -> SchemaIdOf {
+	let identifier = Ss58Identifier::generate(&(&digest).encode()[..])
```
Comment (smohan-dw) [inline pallets/schema/src/mock.rs]: generate is not part of URI anymore. Use Ss58Identifier::to_schema_id()

File: pallets/schema/src/mock.rs [hunk 2]
```diff
@@ -120,7 +120,6 @@ pub mod runtime {
 		type DbWeight = RocksDbWeight;
 		type Version = ();
 		type PalletInfo = PalletInfo;
-		type AccountData = pallet_balances::AccountData<Balance>;
```
Comment (smohan-dw) [inline pallets/schema/src/mock.rs]: can't remove this completely. use type AccountData = ();

File: pallets/schema/src/mock.rs (hunk 1 above)
Comment (vjgaur) [reply pallets/schema/src/mock.rs]: Done

File: pallets/schema/src/mock.rs (hunk 2 above)
Comment (vjgaur) [reply pallets/schema/src/mock.rs]: Done

File: pallets/schema/src/mock.rs [hunk 3]
```diff
@@ -72,107 +75,98 @@ pub mod runtime {
...
+	pub(crate) type TestOwnerOrigin =
+		mock_origin::EnsureDoubleOrigin<TestSchemaPayer, TestSchemaCreator>;
+	pub(crate) type TestOriginSuccess =
+		mock_origin::DoubleOrigin<TestSchemaPayer, TestSchemaCreator>;
+
+	impl pallet_schema::Config for Test {
+		type EnsureOrigin = TestOwnerOrigin;
+		type OriginSuccess = TestOriginSuccess;
+		type RuntimeEvent = RuntimeEvent;
+		type SchemaCreatorId = AccountId32;
```
Comment (smohan-dw) [inline pallets/schema/src/mock.rs]: This assignment should be SubjectId, not AccountId32. This is the reason for origin error

**Pull request closed without merging**